### ISSUE TITLE
App re-register failed

### ISSUE THREAD

**Issue by Breiner214** [MCMrARM/mc-w10-version-launcher#313]
Someone can help me pls?
![imagen_2024-09-01_234052438](https://github.com/user-attachments/assets/ca479031-a251-4780-8b3b-3fff16b2a96a)

**Comment by Breiner214** [comment]
Drops when i launch the version 1.20.1.2

**Comment by Alkrick** [comment]
I get this issue too, I think this occurs because our Windows System's language is not English, is your system not English as well @TotoMako ?
![image](https://github.com/user-attachments/assets/df5a4143-8b44-478b-972e-267418f6a272)

**Comment by TotoMako** [comment]
@Alkrick You are right, my language isnt english on my pc. Probably this could be the issue

**Comment by Alkrick** [comment]
Ok apparently I forgot to turn on developer mode in windows 10 settings as mentioned in the readme, It works fine for me now. If you are trying to launch multiple instances of MC then you need to edit the AppManifest.XML file before launching the game for the first time, editing the file after the game has been launched once doesn't work. If you're not trying to launch multiple instances then ignore this.

**Comment by SheanlR** [comment]
> @Alkrick You are right, my language isnt english on my pc. Probably this could be the issue

I change my language to english it still doesn't work
<img width="736" alt="Screenshot 2024-09-16 231149" src="https://github.com/user-attachments/assets/def61269-19b0-43b6-aeae-855492e843b2">
<img width="461" alt="Screenshot 2024-09-16 231234" src="https://github.com/user-attachments/assets/749a2376-cae4-40eb-9a71-a674893caac5">

**Comment by GreatNateDev** [comment]
This makes me think that since the error is that the temporary directory already exists try a. reinstalling the app b. pressing win+r keys than typing `%TEMP%` than pressing ok and deleting everything in that folder c. restarting you're pc
